app/src/main/cpp/skyline/services/am/applet/ILibraryAppletAccessor.h

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright © 2020 Skyline Team and Contributors (https://github.com/skyline-emu/)
+
+#pragma once
+
+#include <kernel/types/KEvent.h>
+#include <services/base_service.h>
+#include <services/serviceman.h>
+
+namespace skyline::service::am {
+    /**
+     * @brief https://switchbrew.org/wiki/Applet_Manager_services#ILibraryAppletAccessor
+     */
+    class ILibraryAppletAccessor : public BaseService {
+      private:
+        std::shared_ptr<kernel::type::KEvent> stateChangeEvent; //!< This KEvent is triggered when the applet's state changes
+
+      public:
+        ILibraryAppletAccessor(const DeviceState &state, ServiceManager &manager);
+
+        /**
+         * @brief This function returns a handle to applet state change event
+         */
+        void GetAppletStateChangedEvent(type::KSession &session, ipc::IpcRequest &request, ipc::IpcResponse &response);
+
+        /**
+         * @brief This function starts an applet
```

Comment:
"This function starts the applet this service corresponds to (https://switchbrew.org/wiki/Applet_Manager_services#Start)"

Reply:
That's the wrong start